src/dragdrop/core.ts

```diff
@@ -130,11 +130,28 @@ export abstract class DragDropCore<T> extends Base {
   protected createDraggedElementShortcut(text: string, draggedElementNode?: HTMLElement, event?: PointerEvent): HTMLElement {
     const draggedElementShortcut = document.createElement("div");
     draggedElementShortcut.innerText = text;
-    draggedElementShortcut.style.cssText =
-      "height: 24px; min-width: 100px; border-radius: 36px; background-color: white; padding: 16px; cursor: grabbing; position: absolute; z-index: 1000; box-shadow: 0px 8px 16px rgba(0, 0, 0, 0.1); font-family: 'Open Sans'; font-size: 16px; padding-left: 20px; line-height: 24px;";
+    draggedElementShortcut.style.cssText = this.getDraggedElementShortcutStyles();
     return draggedElementShortcut;
   }
 
+  protected getDraggedElementShortcutStyles() {
```

Comment:
Is it possible to define a CSS rule here?
It should support our default color and we should give the ability to override attributes.

Reply:
it make sense I'll fix it

Reply:
fixed